Api/Auth/Policies.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace LearnNet_CatalogService.Api.Auth
+{
+    public static class Policies
+    {
+        public const string Read = "read";
```

Comment:
please do not use the same policy for different resources.
Split each policy into two:
- 'products.read'
- 'categories.read'

and so on